### PR TITLE
#939 enforce REQUIRE_2FA with environment variable (site wide)

### PR DESCRIPTION
**Pull request by mouneyrac**

### CODE REVIEW

File: rdrf/registry/common/middleware.py [hunk 1]
```diff
@@ -30,7 +31,9 @@ def process_request(self, request):
         if user is None or user.is_anonymous:
             return None
 
-        if not user.is_verified() and user.require_2_fact_auth:
+        site_requires_2fa = True if EnvConfig().get("require_2fa", False) == 1 else False
```
Comment (id2359) [inline rdrf/registry/common/middleware.py]: Why not just: site_requires_2fa = EnvConfig().get("require_2fa", False) == 1

Comment (mouneyrac) [reply rdrf/registry/common/middleware.py]: obviously :)

Comment (mouneyrac) [reply rdrf/registry/common/middleware.py]: thanks

Comment (mouneyrac) [reply rdrf/registry/common/middleware.py]: done

**Comment by mouneyrac** [comment]
ready to be merged :)

**Pull request merged**